samples/client/petstore-security-test/swift/SwaggerClient/Classes/Swaggers/AlamofireImplementations.swift

```diff
@@ -66,12 +66,16 @@ class AlamofireRequestBuilder<T>: RequestBuilder<T> {
                         switch v {
                         case let fileURL as NSURL:
                             mpForm.appendBodyPart(fileURL: fileURL, name: k)
+                            break
```

Comment:
these breaks aren't necessary - let's take them out to make the code more readable.

Reply:
Yes I agree. These were in the old script and got generated when I ran it. I just took them out. Thanks for the checking!